Documentation/rpc-errors.md

```diff
@@ -61,8 +61,8 @@ exit status 1
 [status]:       https://godoc.org/google.golang.org/grpc/status#Status
 [new-status]:   https://godoc.org/google.golang.org/grpc/status#New
 [code]:         https://godoc.org/google.golang.org/grpc/codes#Code
-[with-details]: https://godoc.org/google.golang.org/grpc/status#Status.WithDetails
-[details]:      https://godoc.org/google.golang.org/grpc/status#Status.Details
-[status-err]:   https://godoc.org/google.golang.org/grpc/status#Status.Err
+[with-details]: https://godoc.org/google.golang.org/grpc/internal/status#Status.WithDetails
```

Comment:
Thanks for catching this and for the fix!

My concern that this may cause other confusion, because just looking at the godoc, you cannot tell `status.Status` is a type alias of `internal/status.Status`.

Another option is to only keep a link to `status.Status`, but format `status.WithDetails` just as code, and hope users to follow the alias to find the method. WDYT @defawley

Reply:
> My concern that this may cause other confusion, because just looking at the godoc, you cannot tell status.Status is a type alias of internal/status.Status.

You should be able to tell this, as we have a link to the internal package in our godoc. 
https://godoc.org/google.golang.org/grpc/status#Status links to:
> https://godoc.org/google.golang.org/grpc/internal/status

> Another option is to only keep a link to status.Status, but format status.WithDetails just as code, and hope users to follow the alias to find the method.

Either way is OK with me.  It's unfortunate that we have this problem, but we couldn't find any way around it while also avoiding circular dependencies.  Maybe some more creativity could solve that problem?